### PR TITLE
Add documentation for trips

### PR DESCRIPTION
**Pull request by kberzinch**
- Add documentation for trips modeled after dues (final task for #3577)
- Fix some minor UX and documentation bugs
- Remove minimum length requirement for trip purpose

[Rendered docs](https://apiary-test.robojackets.org/docs/)

### CODE REVIEW

File: docs/officers/travel/data-model.rst [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+Trip
+----
+
+A **trip** has a single destination and many travelers.
+
+Trips have a few key attributes:
+
+.. vale write-good.E-Prime = NO
+
+- The **status** which determines whether a trip is visible to travelers
```
Comment (JoshOldenburg) [inline docs/officers/travel/data-model.rst:19]:
```suggestion
- The **status** that determines whether a trip is visible to travelers
```

**Review by JoshOldenburg — CHANGES_REQUESTED** [review]
I'd suggest emphasizing that approval is final and it's "impossible" to go back to draft and change assignments.

File: docs/officers/travel/data-model.rst (hunk 1 above)
Comment (kberzinch) [reply docs/officers/travel/data-model.rst:19]: I fiddled with this bullet point a bit before I opened the PR and I don't really like the current phrasing or your suggestion. This was my other thought, but it doesn't fit the structure of the other bullets.

```suggestion
- The **status** determines whether a trip is visible to travelers
```

Comment (JoshOldenburg) [reply docs/officers/travel/data-model.rst]: I think that's better.

**Review by JoshOldenburg — APPROVED** [review]

**Pull request merged**